etc/kayobe/stackhpc.yml

```diff
@@ -104,15 +104,15 @@ stackhpc_repo_treasuredata_4_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/t
 stackhpc_repo_treasuredata_4_version: "{{ stackhpc_repo_distribution }}"
 
 # Rocky 8 BaseOS
-stackhpc_repo_rocky_baseos_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rocky/8.6/BaseOS/x86_64/os/{{ stackhpc_repo_rocky_baseos_version }}"
+stackhpc_repo_rocky_baseos_url: "{{ stackhpc_repo_mirror_url }}/pulp/content/rocky/8/BaseOS/x86_64/os/{{ stackhpc_repo_rocky_baseos_version }}/"
```

Comment:
Think this is where the CI trips up:

`s|rocky/8|rocky/8.{{ stackhpc_pulp_repo_rocky_8_minor_version }}`

Reply:
In CI terms yes, I think so - that repo has always been available at 8.x rather than 8. Other clients may have a different experience depending on their local configuration...

Reply:
This should all be pretty standard shouldn't it? Or is the ci-aio environment doing something different than a client would?

Reply:
Yeah, I agree - I was just trying to think of how it might've slipped in there. On reflection it probably came from a propagation of the `centos/8/` style URLs above.